product_docs/docs/pgd/5/testingandtuning.mdx

```diff
@@ -93,24 +93,22 @@ transactions.
 
 ### Notes on pgd_bench usage
 
--   When using custom init-scripts it is important to understand implications behind the DDL commands.
-It is generally recommended to wait for the secondary nodes to catch-up on the data-load steps
-before proceeding with DDL operations such as `CREATE INDEX`. The latter acquire global locks which
-can't be acquired until the data-load is complete and thus may time out.
-
--   No extra steps are taken to suppress client messages, such as `NOTICE`s and `WARNING`s emitted
-by PostgreSQL and or any possible extensions including the BDR extension. It is the user's
-responsibility to suppress them by setting appropriate variables (e.g. `client_min_messages`, 
-`bdr.camo_enable_client_warnings ` etc.).
-
+-   When using custom init-scripts, it's important to understand implications behind the DDL commands.
+We generally recommend waiting for the secondary nodes to catch up on the data-load steps
+before proceeding with DDL operations such as `CREATE INDEX`. The latter acquire global locks that
+can't be acquired until the data load is complete and thus might time out.
 
+-   No extra steps are taken to suppress client messages, such as `NOTICE` and `WARNING` messages emitted
+by PostgreSQL and or any possible extensions, including the BDR extension. It's your
+responsibility to suppress them by setting appropriate variables, such as `client_min_messages`, 
+`bdr.camo_enable_client_warnings`, and so on.
 
 ## Performance testing and tuning
 
 PGD allows you to issue write transactions onto multiple master nodes. Bringing
-those writes back together onto each node has a cost in performance.
+those writes back together onto each node has a performance cost.
 
-First, replaying changes from another node has a CPU cost, an I/O cost,
+First, replaying changes from another node has a CPU cost an an I/O cost,
```

Comment:
the original was correct - it's a list of 3 things:
CPU Cost
I/O Cost
Generates WAL records

could say has CPU and I/O cost and generates WAL records